common/djangoapps/student/forms.py

```diff
@@ -115,12 +115,11 @@ class AccountCreationForm(forms.Form):
     validation, not rendering.
     """
     # TODO: Resolve repetition
-    username = forms.SlugField(
+    username = forms.CharField(
```

Comment:
@mattdrayer are you sure that we should move with `CharField` here ? In that way we are allowing user to use any unicode in their `username` .